consensus/chain_selector.go

```diff
@@ -16,6 +16,21 @@ import (
 	"github.com/filecoin-project/go-filecoin/types"
 )
 
+// NewECV is the constant V defined in the EC spec.
+const NewECV uint64 = 2
+
+// PI is the multiplicand in the null penalty term
+const PI float64 = 0.87
+
+// NullThresh is the min number of null rounds before the penalty kicks in
+const NullThresh = 3
+
+// ECV is the constant V defined in the EC spec.
+const ECV uint64 = 10
+
+// ECPrM is the power ratio magnitude defined in the EC spec.
+const ECPrM uint64 = 100
```

Comment:
Don't export these constants. Prefer to collapse into one or a couple of `const (... )` blocks.

Reply:
These don't correspond to the present state of https://filecoin-project.github.io/specs/#chain-weighting, though I believe that is still in flux. I understand that you've implemented the weight function from the pre-October spec (consistent with the rest of the implementation).

That's fine, but I have therefore not mentally verified the spec alignment.

Reply:
Alright. Tradeoff is we have more magic numbers in tests because testing packages can't read them.  I already do a lot of this and am fine with doubling down.